cmd/root.go

```diff
@@ -94,6 +99,40 @@ func init() {
 		os.Exit(1)
 	}
 
+	// Set snapshots dir
+	snapshotsDir = filepath.Join(baseDir, snapshotsDirName)
+
+	// Create snapshots dir if it doesn't exist
+	err = os.MkdirAll(snapshotsDir, os.ModePerm)
+	if err != nil {
+		// no logger here yet
+		fmt.Printf("failed creating the snapshots dir %s: %s\n", snapshotsDir, err)
+		os.Exit(1)
+	}
+
+	// Copy default snapshot if not present
+	defaultSnapshotPath := filepath.Join(snapshotsDir, "anr-snapshot-"+constants.DefaultSnapshotName)
+	if _, err := os.Stat(defaultSnapshotPath); os.IsNotExist(err) {
+		defaultSnapshotBytes, err := ioutil.ReadFile("defaultSnapshot.tar.gz")
```

Comment:
I agree. Shall live outside the repo. Probably for the next release related to pkg management.

Reply:
Really the bootstrap snapshot install shall be downloaded, because for the install script option, we don't have the repo available. Either downloaded+installed from the install script and/or downloaded+installed at first deploy or start time. Will work on the second option for the moment, and download it (for the moment) from the repo.

A note, I think install script shall download avalanchego, evm, snapshot, so first deploy time is much faster, and we conveniently split the waiting times. Also think subnet create command shall download the pluging binary. Same criteria. Finally, install script can also be used in local, so shall replace build script.

Reply:
In this PR the bootstrap snapshot is a resource needed for fast deploy. I see no inconvenient of using it always, but we can decide on that.

Reply:
Other suggestion related to installation. Maybe we can add a hidden command to the CLI to download deps, and make the install script to execute it (instead of duplicating download logic in the script). Just ideas.